components/blocks/fixedColumns.tsx

```diff
@@ -7,34 +7,43 @@ import { componentRenderer } from "./mdxComponentRenderer";
 
 interface FixedColumnsProps {
   data: {
+    header?: string;
     firstColBody: TinaMarkdownContent;
     secondColBody: TinaMarkdownContent;
   };
 }
 
 export const FixedColumns = ({ data }: FixedColumnsProps) => {
   return (
-    <div className="grid md:grid-cols-2 md:gap-6">
-      <div>
-        <TinaMarkdown
-          content={data.firstColBody}
-          components={componentRenderer}
-        />
+    <>
+      {data.header && <h3 className="py-4">{data.header}</h3>}
+      <div className="grid md:grid-cols-2 md:gap-6">
+        <div>
+          <TinaMarkdown
+            content={data.firstColBody}
+            components={componentRenderer}
+          />
+        </div>
+        <div>
+          <TinaMarkdown
+            content={data.secondColBody}
+            components={componentRenderer}
+          />
+        </div>
       </div>
-      <div>
-        <TinaMarkdown
-          content={data.secondColBody}
-          components={componentRenderer}
-        />
-      </div>
-    </div>
+    </>
   );
 };
 
 export const fixedColumnsSchema: Template = {
   name: "FixedColumns",
   label: "Fixed Column Layout (2 columns)",
   fields: [
+    {
+      type: "string",
+      label: "Header",
+      name: "header",
+    },
```

Comment:
Should this be a rich-text field? Or even just moving that specific header to another block instead of this one. It seems odd to include this as a prop.

Reply:
I updated the type to rich-text and renamed it to 'headerSection' to add more meaning. It's beneficial to have it this way because it ensures consistent padding within its fixed layout block.